### PR TITLE
car/interface: retrieve car brand list from car.values.BRANDS

### PR DESCRIPTION
**Pull request by deanlee**
Cache the list of car brands as CAR_BRANDS, improving performance by eliminating redundant directory scans on subsequent function calls.

### CODE REVIEW

File: selfdrive/car/interfaces.py [hunk 1]
```diff
@@ -33,6 +33,8 @@
 TORQUE_OVERRIDE_PATH = os.path.join(BASEDIR, 'selfdrive/car/torque_data/override.toml')
 TORQUE_SUBSTITUTE_PATH = os.path.join(BASEDIR, 'selfdrive/car/torque_data/substitute.toml')
 
+CAR_BRANDS = sorted(entry.name for entry in os.scandir(BASEDIR + '/selfdrive/car') if entry.is_dir())
```
Comment (sshane) [inline selfdrive/car/interfaces.py:36]: Can we not use car.values.BRANDS?

Comment (sshane) [reply selfdrive/car/interfaces.py:48]: How about `__module__`?

Comment (deanlee) [reply selfdrive/car/interfaces.py:48]: Ahhhh, got it!

**Comment by adeebshihadeh** [comment]
I think we just want to get rid of the dynamic importing. Want to open a PR for that?

**Pull request closed without merging**